contrib/msggen/msggen/gen/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .generator import IGenerator, GeneratorChain # noqa
```

Comment:
Aren't these classes which a consumer is going to use should be in a top level `__init__.py` instead of under `gen/` ?
(I'm not very familiar with python packaging, so I might be wrong)

Reply:
This is something that I'm understanding too because works always.


BTW,  from https://stackoverflow.com/a/4116384/10854225 the __init__ is not something related to the module but is something related to the python search module.

>If you remove the __init__.py file, Python will no longer look for submodules inside that directory, so attempts to import the module will fail.


I avoided using the class to import in the `__init__.py` because I would like to keep the code readable, and I like work with the following imports

`from msggen.model import Service`

instead of

`from msggen import Service`

IMHO this make the code much cleaner!